### PR TITLE
Update Camptocamp sub-section

### PR DESCRIPTION
**Pull request by yjacolin**
* Move description to commiters section
* Update description

### CODE REVIEW

**Comment by yjacolin** [comment]
Does PSC member should make a review for such modifications?

File: source/site/forusers/commercial_support.rst [hunk 1]
```diff
@@ -41,6 +41,19 @@ services, and expert training.
 
 ----
 
+.. |camptocamp| image:: /static/site/about/images/camptocamp.png
+   :width: 150 px
+
+|camptocamp| `Camptocamp <http://www.camptocamp.com/en/geospatial-solutions>`_
+(based in France, Germany and Swisszerland)
```
Comment (DelazJ) [inline source/site/forusers/commercial_support.rst]: Switzerland

**Comment by DelazJ** [comment]
> Does PSC member should make a review for such modifications?

Maybe, we could just add at the top of the list a message that description are provided by companies themselves (hence their pov and not QGIS Project's)

**Comment by andreasneumann** [comment]
@yjacolin Is this pull request still meant to be applied?
There seems to be conflicts now, because I applied a newer pull request on the same section first.
So please close this pull request and open a new one, if you still want to apply this modification.
Thanks,
Andreas

**Comment by yjacolin** [comment]
Ah, I forgot this one :) As the last one has just been merged, I will make another PR for the description update :)

Thanks @andreasneumann !

**Pull request closed without merging**